### ISSUE TITLE
Pretrained model for unseen data

### ISSUE THREAD

**Issue by cy94** [WangYueFt/dcp#19]
Hello, 

Thanks for making the code and models public, I was able to reproduce the results of sections 5.1 and 5.3 as they both use the same pretrained model trained on 20 categories.

Could you please upload the pretrained model for section 5.2 - unseen categories as well?

Thanks in advance!

**Comment by Grungeby52** [comment]
@cy94 Hi,  How do I align the two point cloud(stl,ply,...) using dcp? Have you succeeded before?

**Comment by cy94** [comment]
Yes, just follow the instructions given in the README with the pretrained model. To run inference on the whole dataset - 
`python main.py --exp_name=dcp_v2 --model=dcp --emb_nn=dgcnn --pointer=transformer --head=svd --eval --model_path=pretrained/dcp_v2.t7`

You have to modify the Dataloaders and inference code to run it on your own point clouds such as STL or PLY

**Comment by jishuo1** [comment]
Hello.
How did you reproduce the results of Section 5.1 using python main.py --exp_name=dcp_v2 --model=dcp --emb_nn=dgcnn --pointer=transformer --head=svd --eval --model_path=pretrained/dcp_v2.t7?
If I want to train from scratch and don't want to use a pre-trained model, how do I get the same results as in section 5.1?

**Comment by jishuo1** [comment]
@cy94 
Hello.
If I want to train from scratch and don't want to use a pre-trained model, how do I get the same results as in section 5.1?

**Comment by cy94** [comment]
Hi sorry, I don't remember when or how I used this code anymore :)
